docs/guides/src/main/server/proxy.adoc

```diff
@@ -0,0 +1,28 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+
+<@tmpl.guide
+title="Proxy setup"
```

Comment:
IMO, titles should be like `Configuring a Reverse Proxy`.

Reply:
Changed the title, but "Configuring a reverse proxy" is not the right title imo, as we're configuring keycloak to work together with a reverse proxy, not (only) the proxy itself.